view/templates/admin/logs/view.tpl

```diff
@@ -1,6 +1,74 @@
 <div id="adminpage">
 	<h1>{{$title}} - {{$page}}</h1>
-	
+
 	<h3>{{$logname}}</h3>
-	<div style="width:100%; height:400px; overflow: auto; "><pre>{{$data}}</pre></div>
+	{{if $error }}
+		<div id="admin-error-message-wrapper" class="alert alert-warning">
+			<p>{{$error nofilter}}</p>
+		</div>
+	{{else}}
+		<form>
+			<p>
+				<input type="search" name="q" value="{{$q}}" placeholder="search"></input>
+				<input type="Submit" value="search">
+				<a href="/admin/logs/view">clear</a>
+			</p>
+
+
+			<table>
+				<thead>
+					<tr>
+						<th>Date</th>
+						<th>
+							<select name="level" onchange="this.form.submit()">
+								{{foreach $filtersvalues.level as $v }}
+								<option {{if $filters.level == $v}}selected{{/if}} value="{{$v}}">
+									{{if $v == ""}}Level{{/if}}
+									{{$v}}
+								</option>
+								{{/foreach}}
+							</select>
+						</th>
+						<th>
+							<select name="context" onchange="this.form.submit()">
+								{{foreach $filtersvalues.context as $v }}
+								<option {{if $filters.context == $v}}selected{{/if}} value="{{$v}}">
+									{{if $v == ""}}Context{{/if}}
+									{{$v}}
+								</option>
+								{{/foreach}}
+							</select>
+						</th>
+						<th>Message</th>
+					</tr>
+				</thead>
+				<tbody>
+					{{foreach $data as $row}}
+						<tr id="ev-{{$row->id}}" onClick="log_show_details('ev-{{$row->id}}')">
```

Comment:
Please add a visual and text cues that this element can be clicked, through a cursor change and an `alt` attribute.

Reply:
as per MDN `alt` is an attribute for `<img>` tags only.
what about `aria-label`?
with `role="button" tabIndex="0" aria-label="View details" aria-haspopup="true" aria-expanded="false"`, css `cursor:pointer` and js code to toggle `aria-expanded` value and react to "space" and "enter" keypress, should be correctly accessible..

Reply:
This sounds excellent thank you for looking into it.